scripts/index.js

```diff
@@ -1,45 +1,54 @@
+/* eslint-disable comma-dangle */
 /* eslint-disable no-use-before-define */
 /* eslint-disable object-shorthand */
 /* eslint-disable no-undef */
+const booksLocalStorage = JSON.parse(localStorage.getItem('books') || '[]');
```

Comment:
- I can't add books into the array i have this error kindly work on the requested changes:
![Screen Shot 2021-10-19 at 21 36 37](https://user-images.githubusercontent.com/22258274/138018955-7495fd4d-cf95-4886-a4d7-e425bd57d482.png)

Reply:
I am really frustrated with these reviews Its working on my end